vqpy/feat/feat.py

```diff
@@ -121,9 +121,13 @@ def cross_vobj_property(
         ):
     """Decorator for cross-object property computation.
 
-    Wrapper for cross-object property computation. Retrieves list of properties
-    of VObjs of specified type and passes them to the function, as
-    `List[Tuple(property1, property2, ...), ...]`
+    Wrapper for cross-object property computation. Retrieves `cross_vobj_arg`,
```

Comment:
The property function decorated should accept two arguments:
    `self` and `cross_vobj_arg` (positional). During execution, we will pass a list of property values into the second argument (`cross_vobj_arg`). The format of `cross_vobj_arg` is ...

Reply:
Rephrased